sections/featured-collection.liquid

```diff
@@ -30,28 +30,32 @@
-          <a href="{{ section.settings.collection.url }}" class="link underlined-link large-up-hide">{{ 'sections.featured_collection.view_all' | t }}</a>
+  <div class="collection section-{{ section.id }}-padding{% if section.settings.full_width %} collection--full-width{% endif %}">
+      <div class="collection__title title-wrapper title-wrapper--no-top-margin page-width{% if show_slider_on_mobile %} title-wrapper--self-padded-tablet-down{% endif %}">
+        {%- if section.settings.title != blank -%}
+          <h2 class="title {{ section.settings.heading_size }}">{{ section.settings.title | escape }}</h2>
+        {%- endif -%}
+        {%- if section.settings.description != blank or section.settings.show_description -%}
+          <div class="collection__description {{ section.settings.description_style }}">{%- if section.settings.show_description -%}{{ section.settings.collection.description }}{%- else -%}{{ section.settings.description }}{% endif %}</div>
         {%- endif -%}
       </div>
-    {%- endunless -%}
-  
-    <slider-component class="slider-mobile-gutter">
-      <ul id="Slider-{{ section.id }}" class="grid product-grid contains-card{% if settings.card_style == 'standard' %} contains-card--standard{% endif %} grid--{{ section.settings.columns_desktop }}-col-desktop {% if section.settings.collection == blank %} grid--2-col-tablet-down{% else %} grid--{{ section.settings.columns_mobile }}-col-tablet-down{% endif %}{% if show_mobile_slider %} slider slider--tablet grid--peek{% endif %}" role="list">
+
+    <slider-component class="slider-mobile-gutter{% if section.settings.full_width %} slider-component-full-width{% endif%}{% if show_slider_on_mobile == false %} page-width-mobile page-width-tablet{% endif %}{% if show_slider_on_desktop == false and section.settings.full_width == false %} page-width-desktop{% endif %}{% if show_slider_on_desktop %} collection__slider-desktop{% endif %}">
+      <ul id="Slider-{{ section.id }}" class="grid product-grid contains-card{% if settings.card_style == 'standard' %} contains-card--standard{% endif %} grid--{{ section.settings.columns_desktop }}-col-desktop{% if section.settings.collection == blank %} grid--2-col-tablet-down{% else %} grid--{{ section.settings.columns_mobile }}-col-tablet-down{% endif %}{% if show_slider_on_mobile or show_slider_on_desktop %} slider{% if show_slider_on_desktop %} slider--desktop{% endif %}{% if show_slider_on_mobile %} slider--tablet grid--peek{% endif %}{% endif %}" role="list" aria-label="{{ 'sections.featured_collection.slider' | t }}">
         {%- for product in section.settings.collection.products limit: section.settings.products_to_show -%}
-          <li id="Slide-{{ section.id }}-{{ forloop.index }}" class="grid__item{% if show_mobile_slider %} slider__slide{% endif %}">
+          <li id="Slide-{{ section.id }}-{{ forloop.index }}" class="grid__item{% if show_slider_on_mobile or show_slider_on_desktop %} slider__slide{% endif %}">
```

Comment:
This is more a refactor idea than a suggestion for this PR.

We could also restrict `slider__slide` to `.slider .slider__slide`. So we can apply this class anyway without having to repeat the `{% if %}` so much inside of this block. The class would only make any difference when the parent is a `.slider`.